src/components/WalletDropdown/AuthenticatedHeader.tsx

```diff
@@ -177,9 +182,11 @@ const AuthenticatedHeader = () => {
           <USDText>${amountUSD.toFixed(2)} USD</USDText>
         </BalanceWrapper>
         {nftFlag === NftVariant.Enabled && (
-          <ProfileButton onClick={navigateToProfile} size={ButtonSize.medium} emphasis={ButtonEmphasis.medium}>
-            <Trans>View and sell NFTs</Trans>
-          </ProfileButton>
+          <ProfileLink to="/nfts/profile">
```

Comment:
i dont think this is defined behavior. to have a button with a link. just make the button handle routing?